components/uniset/src/provider.rs

```diff
@@ -103,555 +103,40 @@ pub mod key {
         // Enumerated properties
         //
 
-        // Note: The ResourceKey subcategory strings are determined from the Rust enum
-        // integer representations of the Unicode enumerated property name and the
-        // Unicode enumerated property value.
+        // Note: The ResourceKey subcategory strings are determined from the short
+        // alias name of the enumerated property and the short alias name of the
```

Comment:
either "alias" or "name", not "alias name"

Reply:
Done.